cypress/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "project": "./tsconfig.json"
+    "project": "./cypress/tsconfig.json"
```

Comment:
why tsconfig from cypress?

Reply:
this `.eslintrc` is for cypress files and we have seperate `tsconfig.json` for cypress thats why

Reply:
Oh got it, we have two `.eslintrc`